mdf_matio/adapters/generic.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import jsonschema
+import mdf_toolbox
+
+from materials_io.adapters.base import BaseAdapter
+
+
+class GenericMDFAdapter(BaseAdapter):
+    """Generic adapter for MDF extractors. Adapts metadata with MDF-format fields present."""
```

Comment:
I am not sure what this means, so could you spell it out more clearly? My attempt is below:

Generic adapter for metadata produced by parsers originally created by the Materials Data Facility

The parsers originally contributed by the MDF already produce data in formats close to that required by MDF schemas.
The similarity between these parsers output and the MDF schema is therefore large enough to permit a generic adapter for all MDF-derived parsers. 
The parser functions by retrieving the fields from the metadata that are present in the MDF schema.

Reply:
I changed the documentation. Do you think it's sufficient?

Reply:
Yes, it looks sharp!